### PR TITLE
Fix #165, close #161

### LINKED ISSUE #161: Improve check_database_connection function
We can improve this code like that:

```python
def check_database_connection(**kwargs):
        for conn in connections.all():
            try:
                conn.cursor()
                return True
            except OperationalError:
                return False
        return False
```

reported by @eltonplima in #158

### LINKED ISSUE #165: Fix health checker /ping/ - infinite looping

### PR DESCRIPTION
**Pull request by douglasppaz**

### CODE REVIEW

File: bothub/health/checks.py [hunk 1]
```diff
@@ -1,23 +1,38 @@
+import logging
+
+from rest_framework import status
+
+
+logger = logging.getLogger('bothub.health.checks')
+
+CHECK_ACCESSIBLE_API_URL = '/api/repositories/'
```
Comment (eltonplima) [inline bothub/health/checks.py]: Use the python-decouple to get this value from envvar, and let the /api/repositories as default value.

File: bothub/health/checks.py [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
-        return False
-    try:
-        db_conn.cursor()
-        return True
-    except OperationalError as e:
+    if len(connections.all()) is 0:
         return False
+    logger.info('found {} database connection'.format(len(connections.all())))
+    for i, conn in enumerate(connections.all(), 1):
+        db_conn = connections['default']
```
Comment (eltonplima) [inline bothub/health/checks.py]: This code is confuse!

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -9,13 +9,13 @@ services:
...
     environment:
-      - SECRET_KEY
+      - SECRET_KEY=SK
       - DEBUG
       - ALLOWED_HOSTS
       - DEFAULT_DATABASE
       - LANGUAGE_CODE
       - TIME_ZONE
-      - EMAIL_HOST
+      - EMAIL_HOST=
```
Comment (eltonplima) [inline docker/docker-compose.yml]: Only equal?

File: docker/entrypoint.sh [hunk 4]
```diff
@@ -6,7 +6,8 @@ cd $WORKDIR
 python manage.py migrate
 python manage.py collectstatic --noinput
 
-gunicorn -e DJANGO_SETTINGS_MODULE=bothub.settings -b unix:/tmp/bothub.sock -D bothub.wsgi
+gunicorn -e DJANGO_SETTINGS_MODULE=bothub.settings -b unix:/tmp/bothub.sock -D bothub.wsgi --log-file /var/log/gunicorn.log --log-level debug --capture-output
```
Comment (eltonplima) [inline docker/entrypoint.sh]:
```bash
gunicorn -e DJANGO_SETTINGS_MODULE=bothub.settings -b unix:/tmp/bothub.sock -D bothub.wsgi --log-file /var/log/gunicorn.log --log-level ${GUNICORN_LOG_LEVEL:-info} --capture-output
```

And send your log to /dev/stdout

File: docker/entrypoint.sh [hunk 5]
```diff
@@ -6,7 +6,8 @@ cd $WORKDIR
...
 
 mkdir -p /run/nginx/
-nginx -g "daemon off;"
+nginx
+tail -f /var/log/nginx/* /var/log/gunicorn.log
```
Comment (eltonplima) [inline docker/entrypoint.sh]: Please remove these lines and put the old content here

File: docker/docker-compose.yml (hunk 3 above)
Comment (douglasppaz) [reply docker/docker-compose.yml]: yes

**Pull request merged**